python/mkdocs/docs/verify-release.md

```diff
@@ -101,6 +101,6 @@ This will include a Minio S3 container being spun up.
 
 Votes are cast by replying to the release candidate announcement email on the dev mailing list with either `+1`, `0`, or `-1`.
```

Comment:
can we add a "for example" in the end? So we don't really have to update the line below with the latest version anymore